basic-examples/expressions/README.md

```diff
@@ -2,17 +2,25 @@
 
 ## Workflow Description
 
-This workflow demonstrates some example expressions as detailed in our [expression documentation](https://arcalot.io/arcaflow/workflows/expressions/). Arcaflow expressions are a way to reference paths within the workflow or their fields, and/or manipulate the data.
+This workflow demonstrates some example expressions as detailed in our [expression 
+documentation](https://arcalot.io/arcaflow/workflows/expressions/). Arcaflow 
+expressions are a way to reference paths within the workflow or their fields, and they 
+additionally allow you to manipulate the data.
 
-Arcaflow expressions were inspired by JSONPath but have diverged from the syntax. You can use expressions in a workflow YAML like this:
+*Note: Arcaflow expressions were inspired by JSONPath, and may look familar, but 

```

Comment:
Having noticed but not analyzed in detail your exchange here with Webb ... this seems to have degenerated to the point where it's probably misleading and pointless. Anyone who's familiar with JSONPath syntax now knows that Arca expressions are only incidentally similar and they can't trust what they think they know (granted, perhaps there's some small value in that warning, but if that's the direction you want to go the warning should be more explicit) while it adds to someone who isn't familiar with JSONPath you've just thrown out a name and told them it adds no value and should be ignored.

Reply:
Not convinced, but not much concerned, either, so snip ...